apps/src/templates/census2017/CensusInaccuracyReview.jsx

```diff
@@ -0,0 +1,400 @@
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Comment
+                </td>
+                <td>
+                  {this.state.toReview.inaccuracy_comment}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  This Survey's Teaches CS Result
+                </td>
+                <td>
+                  {this.state.toReview.teaches_cs ? 'YES' : 'NO'}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  K8 School?
+                </td>
+                <td>
+                  {this.state.toReview.k8_school ? 'Yes' : 'No'}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  High School?
+                </td>
+                <td>
+                  {this.state.toReview.high_school ? 'Yes' : 'No'}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  How many 10 hours
+                </td>
+                <td>
+                  {this.state.toReview.how_many_10_hours}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  How many 20 hours
+                </td>
+                <td>
+                  {this.state.toReview.how_many_20_hours}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Block programming
+                </td>
+                <td>
+                  {this.state.toReview.topic_blocks ? 'Yes' : 'No'}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Text programming
+                </td>
+                <td>
+                  {this.state.toReview.topic_text ? 'Yes' : 'No'}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <br />
+        <div>
+          <p style={{width: '75%'}}>
+            Explain why we should or should not override the data on this school to say
+            that they <b>{this.state.toReview.teaches_cs ? 'DO' : 'DO NOT'}</b> offer Computer Science:
+          </p>
+          <br />
+          <textarea type="text" style={{height: 100, width: '100%'}} onChange={(event) => {this.setState({notes: event.target.value});}} />
+          <br />
+          {this.state.notes && (
+             <div>
+               <Button
+                 onClick={this.submitInvestigationWithOverride}
+                 size="large"
+                 text={overrideText}
+               />
+               <br />
+               <Button
+                 onClick={this.submitInvestigationWithoutOverride}
+                 size="large"
+                 text="Resolve without override"
+                 style={{marginTop: 5}}
+               />
+             </div>
+          )}
+        </div>
+      </div>
```

Comment:
This is a lot of explicit html here. Can we also generate this table with `Reactabular`?

Reply:
Yep. Done.